packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbSize.stories.tsx

```diff
@@ -31,48 +24,33 @@ const items: Item[] = [
   },
 ];
 
-function renderItem(item: Item, size: BreadcrumbProps['size']) {
-  const isLastItem = items.length - 1 === item.key;
-  return (
-    <React.Fragment key={`${size}-item-${item.key}`}>
-      <BreadcrumbItem current={isLastItem}>{item.value}</BreadcrumbItem>
-      {!isLastItem && <BreadcrumbDivider />}
-    </React.Fragment>
-  );
-}
-
 function renderButton(item: Item) {
   const isLastItem = items.length - 1 === item.key;
   return (
     <React.Fragment key={`button-item-${item.key}`}>
-      <BreadcrumbItem current={isLastItem}>
-        <BreadcrumbButton>{item.value}</BreadcrumbButton>
+      <BreadcrumbItem>
+        <BreadcrumbButton current={isLastItem}>{item.value}</BreadcrumbButton>
       </BreadcrumbItem>
       {!isLastItem && <BreadcrumbDivider />}
     </React.Fragment>
   );
```

Comment:
Similar to https://github.com/microsoft/fluentui/pull/29302#discussion_r1341809280, this should not render a button for the last item right?